soc/common_soc.mk

```diff
@@ -41,7 +41,9 @@ SOFTWARE_ARGS:= --software-load --software-path $(PROJ_DIR)/build/software.bin
 SOC_NAME:=  $(TARGET).$(PROJ)
 OUT_DIR:=   build/$(SOC_NAME)
 UART_ARGS=  --uart-baudrate $(UART_SPEED)
-LITEX_ARGS= --output-dir $(OUT_DIR) --csr-csv $(OUT_DIR)/csr.csv $(CFU_ARGS) $(UART_ARGS) $(TARGET_ARGS) $(EXTRA_LITEX_ARGS)
+LITEX_ARGS= --output-dir $(OUT_DIR) \
+        --csr-csv $(OUT_DIR)/csr.csv --csr-json $(OUT_DIR)/csr.json \
```

Comment:
Do you still need `--csr-csv`? (The same question applies to other `soc/*.mk` files as well).

Reply:
I guess I was thinking that it wouldn't hurt to keep it, and some humans might find it easier to read the csv than the json.   But I guess it could also cause confusion.  I'll check if anything breaks without the csv.